app/objects/c_operation.py

```diff
@@ -128,6 +128,13 @@ def all_relationships(self):
                 if await self._trust_issues(member):
                     break
 
+    async def _active_agents(self):
+        active = []
```

Comment:
move this function down to the private area

Reply:
Ah missed this. Moving now.